src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -540,266 +508,66 @@ private ObjectMetadata syncPutObject(
     public ObjectMetadata storeObject(InputStream object) throws NoSuchAlgorithmException,
```

Comment:
close input stream

Reply:
Thank you! I have added the try-finally blocks to do so.